third_party/srtp/PJSIP_NOTES

```diff
@@ -0,0 +1,28 @@
+Local changes:
+1. Error build "Platform not recognized"
```

Comment:
need to be more specific here. On which platform(s) did it fail?

Reply:
The reason is build error on every platform.
On `datatypes.h` there's a check for certain config. 
(https://github.com/pjsip/pjproject/blob/affc21f07cf610a861b3da82b96e41542b9bb4b4/third_party/srtp/crypto/include/datatypes.h#L57-L63).

The current code doesn't check if those configs are not defined.
https://github.com/pjsip/pjproject/blob/a5023f7fdd4506808e3cc50da2fdbf7b47bdda0b/third_party/srtp/crypto/include/datatypes.h#L57-L63

These configs are defined in `srtp_config.h`. 
Alternatively we can add `#include <srtp_config.h>` to `transport_srtp.c`
https://github.com/pjsip/pjproject/blob/affc21f07cf610a861b3da82b96e41542b9bb4b4/pjmedia/src/pjmedia/transport_srtp.c#L101

This way no changes to datatypes.h is needed